src/test/integration/tests/editor-highlight.js

```diff
@@ -29,7 +29,6 @@ module.exports = async function(ctx) {
   // Wait for the source text to load and make sure we're in the right
   // place.
   await waitForDispatch(dbg, "LOAD_SOURCE_TEXT");
-  assertHighlightLocation(dbg, ctx, "long.js", 66);
```

Comment:
same as below. the test is temporarily duplicated because the web version was more intermittent in debug builds...